### ISSUE TITLE
Discovery Metadata Techniques for Fixed Layout

### ISSUE THREAD

**Issue by gregoriopellegrino** [w3c/epub-specs#2908]
### Section

Discovery Metadata Techniques for Fixed Layout

### Describe the problem

The section is empty and refers to generic references.

### Describe the fix or new feature you propose

I suggest moving the section of the "Best Practices" document [Package metadata](https://www.w3.org/TR/epub-fxl-a11y/#package-metadata) in the "Fixed Layout Accessibility Techniques" document to the section "Discovery Metadata Techniques for Fixed Layout".

**Comment by clapierre** [comment]
So, I agree sections
3.1 Using manifest fallbacks in fixed layout EPUBs
3.2 Examples of accessibility metadata
Should be moved from the main "EPUB Accessibility - Fixed Layout Challenges and Best Practices" document.

But I think perhaps the main section 3 Package metadata should stay with the main document and after add a link to "EPUB Accessibility - Fixed Layout Techniques"
2. Discovery Metadata Techniques for Fixed Layout
 
where the original 3.1 & 3.2 from the best practices, will become 2.1 & 2.2 in our techniques document.

@wareid and @gregoriopellegrino do you all agree?  If so I can make a PR moving that over.
